.stylelintrc.js

```diff
@@ -4,23 +4,22 @@ module.exports = {
     // TypeScript declaration files contain no styles.
     // Stylelint is also reporting parseError on `docs/types/react-docgen.d.ts`.
     '**/*.d.ts',
+    'docs/.next/**/*',
+    'docs/export/**/*',
+    '**/node_modules/**/*',
   ],
   rules: {
     'alpha-value-notation': null,
     'custom-property-pattern': null,
-    'declaration-colon-newline-after': null,
-    'function-parentheses-newline-inside': null, // not compatible with prettier
     'media-feature-range-notation': null,
     'no-empty-source': null,
-    'no-missing-end-of-source-newline': null,
     'selector-class-pattern': null,
     'string-no-newline': null, // not compatible with prettier
     'value-keyword-case': null,
-    'value-list-comma-newline-after': null, // not compatible with prettier
```

Comment:
No idea what's going on here. I'm simply trying to follow the default as much as possible, it seems to pass. I saw Base UI is doing this.